nginx/nginx.conf.d/coala.io.conf

```diff
@@ -25,7 +25,16 @@ server {
     location /rebase    { return 302 https://coala.io/git#rebasing; }
     location /commit    { return 302 http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html; }
     location /cep       { return 302 https://github.com/coala/cEPs/blob/master/cEP-0000.md; }
+    location /cep1      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0001.md; }
+    location /cep2      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0002.md; }
+    location /cep3      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0003.md; }
+    location /cep5      { return 302 https://github.com/coala/cEPs/blob/master/cEP-0005.md; }
     location /coc       { return 302 https://github.com/coala/cEPs/blob/master/cEP-0006.md; }
```

Comment:
I would also add a `cep6` synonym too, so people don't wonder why just this one doesn't work, they are probably not aware of `coc` then^^